docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,26 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin.
+This guide explains how to build a logs data source plugin. 
```

Comment:
Consider deleting this sentence duplicates the page title.

Reply:
Done. I will check in this fix soon.

Reply:
Check in fix, available for review.